our_package/test/data/test_data.csv

```diff
@@ -0,0 +1,102 @@
+ time   norm   <x>   <y>   <z>   <H>
```

Comment:
this is a test file for comparison and should live in test/data - later when publishing the package you can also include data files in the distribution like this